lib/theme-options.php

```diff
@@ -119,6 +119,31 @@ function add_options_page_metabox() {
 			'type' => 'checkbox',
 		) );
 
+		// SPLASH
+
+		$options_metabox->add_field( array(
```

Comment:
o this is the title thing. its like a title for the metabox display not an actual field